### ISSUE TITLE
Go sur la zone

### ISSUE THREAD

**Issue by tcharlss** [nursit/factures#6]
Maintenant qu'on a des dépôts qui peuvent être privés sur la zone, est-ce que ce plugin pourrait être déplacé dessus ?

Ça simplifierait le suivi des tickets, demandes d'ajout et cie pour tout le monde.

Le problème c'est l'ancien plugin [factures](https://git.spip.net/spip-contrib-extensions/factures) obsolète qui a le même préfixe.
C'est jouable ce cas de figure ? De façon à ce que svp et le débardeur ne s'emmêlent pas les pinceaux ?

**Comment by rastapopougros** [comment]
Oui, une orga (à priori puisque vous avez déjà tous des comptes persos) git.spip.net/nursit/ permettrait de faire un dépôt git.spip.net/nursit/factures et profiter de tous les outils, empaquetage, trad, etc plus facilement

**Comment by pierretux** [comment]
Un retour sur cette demande ? Cela serait bien pratique

**Comment by Cerdic** [comment]
Les outils marchent pareil avec github, le vrai problème c'est la collision de préfixe avec le vieux vieux plugin factures. Du coup j'ai avancé en déplaçant le vieux plugin factures de la zone dans une organisation spip-grenier
https://git.spip.net/spip-grenier/factures

**Comment by rastapopougros** [comment]
Ce n'est pas une question d'outils d'empaquetage (qui oui marche avec github), comme l'explique @tcharlss en premier, ça simplifie le suivi de tout ce qu'on utilise de SPIP dans un même outil au même endroit : pour suivre les tickets, faire les PR etc, avec le même compte utilisateur, pas dans d'autres outils ailleurs. Et cela est possible sans problème en gardant la main dans une orga sans laisser les commits à tout le monde.

**Comment by pierretux** [comment]
Du coup on peut avoir ce plugins factures dans les flux svp maintenant ?
